### PR TITLE
dotCMS/core#14495 Update colors in the jsps

### PR DESCRIPTION
**Pull request by fmontes**

### CODE REVIEW

File: src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts [hunk 1]
```diff
@@ -20,15 +20,16 @@ export class DotUiColorsService {
      * @param {DotUiColors} colors
      * @memberof DotUiColorsService
      */
-    setColors(colors: DotUiColors): void {
-        this.html = document.querySelector('html');
-        this.setColorMain(colors.primary);
-        this.setColorSec(colors.secondary);
-        this.setColorBackground(colors.background);
+    setColors(el: HTMLElement, colors?: DotUiColors): void {
```
Comment (freddyucv) [inline src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts]: doc

File: src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts [hunk 2]
```diff
@@ -42,32 +43,32 @@ export class DotUiColorsService {
         return `${rgb.r}, ${rgb.g}, ${rgb.b}`;
```
Comment (freddyucv) [inline src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts]: the values into this.currentColors and window.getComputedStyle don't should be the same?

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts [hunk 3]
```diff
@@ -43,6 +43,7 @@ import { DotContentletEditorService } from '../../../view/components/dot-content
 import { DotPageContainer } from '../shared/models/dot-page-container.model';
 import { DotEditContentComponent } from './dot-edit-content.component';
 import { DotContentletEditorModule } from '../../../view/components/dot-contentlet-editor/dot-contentlet-editor.module';
+import { DotUiColorsService } from '../../../api/services/dot-ui-colors/dot-ui-colors.service';
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts]: why do we need this import?

File: src/app/portlets/dot-edit-page/layout/dot-edit-layout-advanced/dot-edit-layout-advanced.component.spec.ts [hunk 4]
```diff
@@ -38,6 +39,7 @@ const basicModule = {
     declarations: [DotEditLayoutAdvancedComponent],
     providers: [
         DotGlobalMessageService,
+        DotUiColorsService,
```
Comment (freddyucv) [inline src/app/portlets/dot-edit-page/layout/dot-edit-layout-advanced/dot-edit-layout-advanced.component.spec.ts]: why do we need provide this service here?

File: src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts (hunk 2 above)
Comment (fmontes) [reply src/app/api/services/dot-ui-colors/dot-ui-colors.service.ts]: No necessarily, `this.currentColors` is updated when you call `setColors` with a color param, then we applied to the and then you have the same, so at some point you don't have the same.

**Pull request merged**